npm/src/lib/node.js

```diff
@@ -30,6 +30,13 @@ export const fs = {
       });
     });
   },
+  readFile: function(f: string, opts: Object): Promise<Buffer> {
```

Comment:
How come you chose to write this node file, instead of using something like promisifyAll from Bluebird? 

http://bluebirdjs.com/docs/api/promise.promisifyall.html